### ISSUE TITLE
feat: (YouTube) Remove or auto-skip member's only videos

### ISSUE THREAD

**Issue by OkamiTakahashi** [ReVanced/revanced-patches#2550]
### Feature description

As the title states. It's pretty self explanitory, and I don't get why there's two description boxes here as they can be rolled into a singular one, but anyway, members only videos from my experience wind up in my playlists because for whatever reason, ASMRtists feel compelled to change their older videos to members only, prompting users to pay to continue. This is a very annoying thing to me. There is no way to remove them from the playlist while said playlist is in use, nor is there a way to skip them, it's literally just stuck there until you pay them money. I just wanna  sleep,man, don't interrupt my sleep because you suddenly want money.

### Motivation

As stated above, this box is useless as the movitation and feature descriptions go hand in hand; they're intertwined so these boxes should be rolled into one, but to repeat myself: it's for ease of access. The playlist is perma-stuck until you exit the playlist or pay money to join a membership. While the playlist is in use, I can't remove the now member's only video(s) from my playlist as none of the regular buttons for sharing, adding to playlists, etc, show up when the playlist is stuck on said member's only video. There is also no way to skip it automatically to go to the next video. I think I'm not the only one who could be experiencing this and I think overall it will be beneficial to those who used ReVanced as a way to enjoy their ASMR playlist without being woken up or interrupted by inspid ads.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Just dont add member only videos to your playlist then?

**Comment by OkamiTakahashi** [comment]
No, it definitely isn't for me. And I don't deliberate add member's only videos to my playlists, the video uploaders are obviously changing the status TO members' only after I've already added it to my playlist.

I personally think that you are making a mistake closing this. It may not seem significant or important to you, but it certainly is. Come on, think about it. How inconvenient is it for a playlist to be stuck on a member's only video that can't be skipped over?You have to exit the playlist and reshuffle. Playlists like this get GINORMOUS and having to scour through them to manually remove them is a pain in the ass. When I posted about this on the sub, other users agreed with me and recommended I come here to post the suggestion. I must say, I'm very disappointed. Thanks for wasting my time.

**Comment by eblieb** [comment]
I would love to see this added ...
